doc/index.html

```diff
@@ -23,8 +23,8 @@
 <body>
 
 <div class="header">
-  <h1>Synopsys Detect</h1>
-  <p>Redirecting to latest Synopsys Detect documentation.</p>
+  <h1>Blackduck Detect</h1>
```

Comment:
New naming scheme is "Black Duck Detect". ; )

Reply:
It should be just Detect.

Reply:
We should use the official product name here, "Black Duck Detect".

We can/should use just "Detect" in cases where the full product name has already been defined, but initial/main references should be the full name.